src/modules/mim/deliveryoptimization.json

```diff
@@ -0,0 +1,103 @@
+{
+  "name": "DeliveryOptimizationModel",
+  "type": "mimModel",
+  "contents": [
+    {
+      "name": "DeliveryOptimization",
+      "type": "mimComponent",
+      "contents": [
+        {
+          "name": "desiredDeliveryOptimizationPolicies",
+          "type": "mimObject",
+          "desired": true,
+          "schema": {
+            "fields": [
+              {
+                "name": "percentageDownloadThrottle",
+                "schema": "integer"
+              },
+              {
+                "name": "cacheHostSource",
+                "schema": {
+                  "type": "enum",
+                  "valueSchema": "integer",
+                  "enumValues": [
+                    {
+                      "name": "none",
+                      "enumValue": 0
+                    },
+                    {
+                      "name": "azureDeviceUpdate",
+                      "enumValue": 1
+                    },
+                    {
+                      "name": "deviceDiscoverySerivce",
```

Comment:
This is not spelled right. It should be `deviceDiscoveryService`, not `deviceDiscoverySerivce`.

Reply:
Same for reported version.